tensorflow/workspace2.bzl

```diff
@@ -201,8 +201,13 @@ def _tf_repositories():
     tf_http_archive(
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
-        patch_file = ["//third_party/mkl_dnn:onednn_acl_threadcap.patch", "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch"],
-        sha256 = "fc2b617ec8dbe907bb10853ea47c46f7acd8817bc4012748623d911aca43afbb",
+        patch_file = [
+	"//third_party/mkl_dnn:onednn_acl_threadcap.patch", 
+	"//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", 
+	"//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch", 
+	"//third_party/mkl_dnn:onednn_acl_threadpoolscheduler.patch"
+		],
```

Comment:
I think the formatting here looks weird, but let's see if the formatter presubmit will complain.

Reply:
Okay!

Reply:
Is there anything that needs to be changed?

Reply:
No. It got a conflict so I manually fixed it internally today. Just waiting for morning PT so other folks could help approve my changes. (I renamed the patch to `onednn_acl_threadpool_scheduler.patch`, btw.) Thank you for checking!